modules/core/src/shaderlib/project/viewport-uniforms.ts

```diff
@@ -177,8 +177,8 @@ function calculateMatrixAndOffset(
   }
 
   return {
-    viewMatrix,
-    viewProjectionMatrix,
+    viewMatrix: viewMatrix as NumArray16,
```

Comment:
In the interest of avoiding cases, I thought we'd declare NumArray16 as `[number, ...] | Matrix4x4` so we could avoid this? Or should we add a method to the matrix that casts it to NumArray16?

Reply:
That won't help here as `viewMatrix` is `number[]`. The better fix would be to update `viewport.ts` to use `viewMatrix?: `NumArray16` limit the scope of this work

Reply:
Understood. I just put up a PR with more specific types in math.gl. If we gain enough confidence around what the best approach is we can start rolling these out in the frameworks, doing this bottom up so to speak.